spec/http_cache_spec.rb

```diff
@@ -23,7 +23,7 @@
   end
 
   it 'logs that a POST request is unacceptable' do
-    expect(logger).to receive(:debug).with('HTTP Cache: [POST /post] unacceptable, delete')
+    expect(logger).to receive(:debug) { |&block| expect(block.call).to eq 'HTTP Cache: [POST /post] unacceptable, delete' }
```

Comment:
Could you please wrap the arguments of each `eq` matcher with parenthesis? Just to keep consistent with the rest of the matchers in this file.